Cubical/Categories/Category.agda

```diff
@@ -24,59 +24,90 @@ module Cubical.Categories.Category where
 open import Cubical.Core.Glue
 open import Cubical.Foundations.Prelude
 
+private
+  variable
+    ℓ ℓ' : Level
+
 -- Precategories
 
 record Precategory ℓ ℓ' : Type (ℓ-suc (ℓ-max ℓ ℓ')) where
   no-eta-equality
   field
     ob : Type ℓ
-    hom : ob → ob → Type ℓ'
-    idn : ∀ x → hom x x
-    seq : ∀ {x y z} (f : hom x y) (g : hom y z) → hom x z
-    seq-λ : ∀ {x y : ob} (f : hom x y) → seq (idn x) f ≡ f
-    seq-ρ : ∀ {x y} (f : hom x y) → seq f (idn y) ≡ f
-    seq-α : ∀ {u v w x} (f : hom u v) (g : hom v w) (h : hom w x) → seq (seq f g) h ≡ seq f (seq g h)
+    Hom[_,_] : ob → ob → Type ℓ'
+    id : ∀ x → Hom[ x , x ]
+    _⋆_ : ∀ {x y z} (f : Hom[ x , y ]) (g : Hom[ y , z ]) → Hom[ x , z ]
+    seq-λ : ∀ {x y : ob} (f : Hom[ x , y ]) → (id x) ⋆ f ≡ f
```

Comment:
I would change these names as well. How about `⋆IdL`, `⋆IdR` and `⋆Assoc`? 

This should be in line with what we are doing in Algebra. Or what does @felixwellen think?

Reply:
Yes, that fits the style of the (new) names in Algebra.

Reply:
changed